### ISSUE TITLE
Create a logo

### ISSUE THREAD

**Issue by furilo** [TrackDons/trackdons#80]

**Comment by bomberstudios** [comment]
Took almost a year, but I'm working on this during the holidays : )

**Comment by furilo** [comment]
woooa! ;)

2016-01-04 14:02 GMT+01:00 Ale Muñoz <notifications@github.com>:

> Took almost a year, but I'm working on this during the holidays : )
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/TrackDons/trackdons/issues/80#issuecomment-168672383>.
>



-- 
http://twitter.com/furilo
http://www.furilo.com

**Comment by bomberstudios** [comment]
An idea I'm working with is the concept of recurrence and love (since you basically support what matters to you):

![heart frame green](https://cloud.githubusercontent.com/assets/3832/12112450/e451a290-b39c-11e5-88dc-4b8c811a8b1a.png)

![heart frame white](https://cloud.githubusercontent.com/assets/3832/12112451/e556578a-b39c-11e5-9d00-ad03738ffc1a.png)

I toyed with the idea of adding "money" to the mix:

![heart frame white dollar](https://cloud.githubusercontent.com/assets/3832/12112601/dcfab526-b39d-11e5-92dc-33cf9ab42f6e.png)

but I like my logos subtle and that reference is too obvious and inelegant for my taste (also: showing just one currency symbol is a minefield in a global economy : )

If we wanted to add the concept of "track" we could do something like this:

![loupe](https://cloud.githubusercontent.com/assets/3832/12112903/b8c780ce-b39f-11e5-8769-a216bda02a23.png)

but it ends up looking too busy, and since it's already on the name, I think I'd stick with the simpler version (type is for positioning only, haven't really spent time working on it):

![heart frame type](https://cloud.githubusercontent.com/assets/3832/12113030/8aea4d7a-b3a0-11e5-9d05-18430572bbf5.png)

Your comments, please? : )

**Comment by nobuti** [comment]
Enorme.

**Comment by furilo** [comment]
Love it! +1 to the simpler version (also think that the currency symbol is not needed)

btw, open to using new colors (and really, anything about the design ;)

**Comment by bomberstudios** [comment]
Glad to see you like it! : )

Tiny versions to see how it works when scaled (16px and 32px, 16px is not really pixel-optimised):

![heart 16px](https://cloud.githubusercontent.com/assets/3832/12113300/2760f572-b3a2-11e5-813e-8409675d4b8c.png)

![heart 32px](https://cloud.githubusercontent.com/assets/3832/12113301/2840c0d0-b3a2-11e5-9bc0-508f7d5ef55a.png)

About colours, I was working with reds initially, so I'll keep exploring that front:

![heart](https://cloud.githubusercontent.com/assets/3832/12113369/83affb34-b3a2-11e5-914a-dedb10b8f9b7.png)

**Comment by bomberstudios** [comment]
Been tweaking this for a while, and I keep coming back to the original:

![logo-band-white 2x](https://cloud.githubusercontent.com/assets/3832/12202807/f8af953a-b62b-11e5-8d05-f4c1b18c07c0.png)

![logo-band 2x](https://cloud.githubusercontent.com/assets/3832/12202812/fde2dada-b62b-11e5-92cb-6f536f6890cc.png)

That's #CF0000 + #A50000, and Roboto Regular.

If we're ok with this, I'll export some assets and add them to the repo (and will use them on the site : )

**Comment by furilo** [comment]
seeing again the red and the heart, maybe it reminds too much of health?

2016-01-08 17:22 GMT+01:00 Ale Muñoz <notifications@github.com>:

> Been tweaking this for a while, and I keep coming back to the original:
>
> [image: logo-band-white 2x]
> <https://cloud.githubusercontent.com/assets/3832/12202807/f8af953a-b62b-11e5-8d05-f4c1b18c07c0.png>
>
> [image: logo-band 2x]
> <https://cloud.githubusercontent.com/assets/3832/12202812/fde2dada-b62b-11e5-92cb-6f536f6890cc.png>
>
> That's #CF0000 + #A50000, and Roboto Regular.
>
> If we're ok with this, I'll export some assets and add them to the repo
> (and will use them on the site : )
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/TrackDons/trackdons/issues/80#issuecomment-170046040>.
>



-- 
http://twitter.com/furilo
http://www.furilo.com

**Comment by bomberstudios** [comment]
Yeah, now you mention it the red badge kinda looks like a counter for a blood donation campaign : )

Here's green:

![logo-band-white 2x](https://cloud.githubusercontent.com/assets/3832/12203058/4e1104cc-b62d-11e5-9d69-086ef968f0ac.png)

![logo-band 2x](https://cloud.githubusercontent.com/assets/3832/12203059/4e426166-b62d-11e5-9d0b-ca5db020b9ee.png)

and (you knew I would do this) orange:

![logo-band-white 2x](https://cloud.githubusercontent.com/assets/3832/12203158/c2b1705a-b62d-11e5-94fb-4f5e21bcadff.png)
![logo-band 2x](https://cloud.githubusercontent.com/assets/3832/12203159/c2e264b2-b62d-11e5-9304-ec744cfb6247.png)

**Comment by furilo** [comment]
orange looks good, no problem in changing it. buy changing all greens in
the site for the oranges, without changing anything else maybe will produce
a strange result?

2016-01-08 17:32 GMT+01:00 Ale Muñoz <notifications@github.com>:

> Yeah, now you mention it the red badge kinda looks like a counter for a
> blood donation campaign : )
>
> Here's green:
>
> [image: logo-band-white 2x]
> <https://cloud.githubusercontent.com/assets/3832/12203058/4e1104cc-b62d-11e5-9d69-086ef968f0ac.png>
>
> [image: logo-band 2x]
> <https://cloud.githubusercontent.com/assets/3832/12203059/4e426166-b62d-11e5-9d0b-ca5db020b9ee.png>
>
> and (you knew I would do this) orange:
>
> [image: logo-band-white 2x]
> <https://cloud.githubusercontent.com/assets/3832/12203158/c2b1705a-b62d-11e5-94fb-4f5e21bcadff.png>
> [image: logo-band 2x]
> <https://cloud.githubusercontent.com/assets/3832/12203159/c2e264b2-b62d-11e5-9304-ec744cfb6247.png>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/TrackDons/trackdons/issues/80#issuecomment-170048537>.
>



-- 
http://twitter.com/furilo
http://www.furilo.com

**Comment by bomberstudios** [comment]
Let me make a quick mockup of how it could look like (we don't need to make the whole site orange, and we could in fact keep it green : )
